nano/node/node.hpp

```diff
@@ -163,6 +164,7 @@ class node final : public std::enable_shared_from_this<nano::node>
 	nano::network_params network_params;
 	nano::node_config config;
 	nano::stat stats;
+	std::shared_ptr<nano::websocket::payment_validator> websocket_payment_validator;
```

Comment:
Is it necessary to store this here? Seems it can just be created and passed to `websocket_server` which then has a handle to it. And probably can be `std::unique_ptr` there too.

Reply:
Gone through the node ref removal refactoring